### ISSUE TITLE
Feature request: Linkify release notes if it's a URL

### ISSUE THREAD

**Issue by NickCraver** [NuGetPackageExplorer/NuGetPackageExplorer#1309]
**Type** (choose one): Feature request
**NPE version**: 5.10.43
**OS version**: Windows 10
**Installed from**: Microsoft Store

It'd be handy if a few items, if they are links, were linkified in the left-hand panel. Today we see this:

![Release Notes](https://user-images.githubusercontent.com/454813/123552519-7aa03600-d744-11eb-9f10-e6df115ed82b.png)
(note that the text is not selectable)

If the content is a URL, can it please be a link like other items? If it helps, the NuGet gallery [also does this](https://github.com/NuGet/NuGetGallery/pull/4505) and it's pretty handy :)

**Comment by clairernovotny** [comment]
Any reliable ways to detect URLs from the string? A good regex, perhaps? :)

**Comment by NickCraver** [comment]
Yup, I recommend the one the gallery is using for 1:1, taken from Stack Overflow at some point =P Linky: https://github.com/NuGet/NuGetGallery/blob/main/src/NuGetGallery/Helpers/HtmlExtensions.cs#L129

**Comment by campersau** [comment]
What about `Uri.TryParse`?

**Comment by clairernovotny** [comment]
@campersau depends on how the feature is implemented. If the assumption is that it's either text or a URL, that could work. If there's a URL in the middle of some text, it'd need a RegEx.
